### ISSUE TITLE
Release 16.1.0

### ISSUE THREAD

**Issue by ybiquitous** [stylelint/stylelint#7421]
Waiting for review:

- [x] #7416
- [x] #7417
- [x] #7423

---

- [x] stylelint release
- [x] stylelint-config-recommended update
- [x] stylelint-config-standard release
  - https://github.com/stylelint/stylelint-config-standard/pull/306
- [x] stylelint.io update
- [x] stylelint-demo check
- [x] tweet

We will add a new rule [`lightness-notation`](https://github.com/stylelint/stylelint/tree/main/lib/rules/lightness-notation) (#7366), which will be included in our standard config.

For other changes, see the [ongoing changelog](https://github.com/stylelint/stylelint/pull/7415/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed). Also, see commits: https://github.com/stylelint/stylelint/compare/16.0.2...main

**Comment by jeddy3** [comment]
Waiting on those PRs SGTM.

We'll also need a PR to mark `lightness-notation` in [docs/user-guide/rules.md](https://stylelint.io/user-guide/rules).

**Comment by ybiquitous** [comment]
> We'll also need a PR to mark `lightness-notation` in [docs/user-guide/rules.md](https://stylelint.io/user-guide/rules).

Already done?

https://github.com/stylelint/stylelint/blob/19ab06a4ca55da15ff64a6391037a47a01136229/docs/user-guide/rules.md?plain=1#L339

**Comment by ybiquitous** [comment]
I've just added #7423 to the review-waiting list.

---

@mattxwang How about #7416? Can we include it in 16.1.0?

**Comment by mattxwang** [comment]
> @mattxwang How about #7416? Can we include it in 16.1.0?

Sounds good to me!

**Comment by ybiquitous** [comment]
I will start the release today.

**Comment by ybiquitous** [comment]
16.1.0 is out, and all tasks are done! 🎉 

Thank you so much to everyone who contributed to this release! 😊
